README.md

```diff
@@ -37,16 +37,17 @@ available.
 A simplistic regex-based implementation can be found in the `test` folder. This
 implementation has 100% compliance to the test suite.
 
-[Open Census](https://opencensus.io) has implementations for the following
+[Open Telemetry](https://opentelemetry.io) has alpha implementations for the following
 languages:
 
-1. [C#](https://github.com/census-instrumentation/opencensus-csharp/blob/4a8ddf6727eafda97a06c7c30d8a4fc2ec8b8e2f/src/OpenCensus/Trace/Propagation/TraceContextFormat.cs)
-2. [Erlang](https://github.com/census-instrumentation/opencensus-erlang/blob/b3ab781b060b15a3cacbf43717c3aeb0c90c4a08/src/oc_propagation_http_tracecontext.erl)
-3. [Go](https://github.com/census-instrumentation/opencensus-go/blob/ae11cd04b7789fa938bb4f0e696fd6bd76463fa4/plugin/ochttp/propagation/tracecontext/propagation.go)
-4. [Java](https://github.com/census-instrumentation/opencensus-java/blob/e5e9d9224a1c9c5ee981981cf29e86662aef08c6/impl_core/src/main/java/io/opencensus/implcore/trace/propagation/TraceContextFormat.java)
-5. [Node.js](https://github.com/census-instrumentation/opencensus-node/blob/fa97a9b6f19b97e1038ffa9e1be4b407f3844df2/packages/opencensus-propagation-tracecontext/src/tracecontext-format.ts)
-6. [Python](https://github.com/census-instrumentation/opencensus-python/blob/2aef803e4a786fe0ffb14b168a8458283ccd72a0/opencensus/trace/propagation/trace_context_http_header_format.py)
-7. [Ruby](https://github.com/census-instrumentation/opencensus-ruby/blob/8cb9771b218e440e825c99981ea405d40f735926/lib/opencensus/trace/formatters/trace_context.rb)
+1. [.NET](https://github.com/open-telemetry/opentelemetry-dotnet/blob/605760cf5ae0511b94d475a754f5ca1a5ec35f41/src/OpenTelemetry.Api/Context/Propagation/TraceContextFormat.cs)
+2. [Go](https://github.com/open-telemetry/opentelemetry-go/blob/15bfc5bb12a4e0a02b1b96b701fd2881cefe1158/propagation/http_trace_context_propagator.go)
+3. [JavaScript](https://github.com/open-telemetry/opentelemetry-js/blob/4f48357b7d1dea246a67ef1a0e071fc64aa10e26/packages/opentelemetry-core/src/context/propagation/HttpTraceContext.ts)
+4. [Java](https://github.com/open-telemetry/opentelemetry-java/blob/0b21928db5085fe3deff6dfd8d28bd49dcde9e71/api/src/main/java/io/opentelemetry/trace/propagation/HttpTraceContext.java)
+5. [Python](https://github.com/open-telemetry/opentelemetry-python/blob/602d42a45f6a4684342b298c5a7c4dab680e301a/opentelemetry-api/src/opentelemetry/context/propagation/tracecontexthttptextformat.py)
+6. [Ruby](https://github.com/open-telemetry/opentelemetry-ruby/blob/bd6baeebb2fe65ed22e9ad33920387ce0ee8870f/api/lib/opentelemetry/distributed_context/propagation/trace_parent.rb)
+
+Open Telemetry implementations are in active development for C++, Erlang, and PHP. In addition, [Open Census](https://opencensus.io) has an implementation for [Erlang](https://github.com/census-instrumentation/opencensus-erlang/blob/b3ab781b060b15a3cacbf43717c3aeb0c90c4a08/src/oc_propagation_http_tracecontext.erl).
```

Comment:
```suggestion
OpenTelemetry is the next version of OpenCensus. OpenTelemetry implementations are in active development for C++, Erlang, and PHP. In addition, [Open Census](https://opencensus.io) has an implementation for [Erlang](https://github.com/census-instrumentation/opencensus-erlang/blob/b3ab781b060b15a3cacbf43717c3aeb0c90c4a08/src/oc_propagation_http_tracecontext.erl) and the languages listed above.
```

Reply:
I don't think it's relevant to this context.

Reply:
It's also not really accurate as OpenTelemetry is a replacement for OpenCensus and OpenTracing, and is not really the next version of either.